src/kOS/Utilities/VesselUtils.cs

```diff
@@ -227,6 +227,11 @@ public static Vector3d GetNorthVector(Vessel vessel)
 
         public static object TryGetEncounter(Vessel vessel, SharedObjects sharedObj)
         {
+            // If not the active vessel, it will be on rails, and therefore won't
+            // be able to have "encounters" via its patchedConicSolver.
+            if (vessel.patchedConicSolver == null)
+                return "None";
+
```

Comment:
I never liked the fact that the ENCOUNTER suffix only finds *maneuver node* encounters and can't find just natural encounters that you'll hit by momentum from where you are now.  (the way you'd do that is to follow the vessel's orbit object, which has a next orbit linked list structure to it to represent future non-manuever node encounters).

But as the "remit" for this PR is JUST to fix the crashes from patchedConicSolver being null, that fell outside the remit of this change.

It might make a good separate issue.